test/boost/tablets_test.cc

```diff
@@ -132,7 +132,7 @@ SEASTAR_TEST_CASE(test_tablet_metadata_persistence) {
                         tablet_replica {h3, 1},
                     },
                     db_clock::now(),
-                    locator::tablet_task_info::make_auto_repair_request(),
+                    locator::tablet_task_info::make_auto_repair_request("", "dc1,dc2"),
```

Comment:
Could you explain why  the default no dc host selection is not enough?

Reply:
It is enough. When adding previous changes to system.tablets (e.g. resize_task_info) I was told to add it to this test to make sure it will be persisted. I thought it's worth doing here, too.